test/feature.go

```diff
@@ -71,31 +77,38 @@ func parsePath(path string) ([]*gherkin.Feature, error) {
 		return parseFeatureDir(path)
 	}
 
-	ft, err := parseFeatureFile(path)
+	newIDFunc := (&messages.Incrementing{}).NewId
+	ft, err := parseFeatureFile(path, newIDFunc)
 	if err != nil {
 		return features, err
 	}
 
 	return append(features, ft), nil
 }
 
-func parseFeatureFile(path string) (*gherkin.Feature, error) {
+func parseFeatureFile(path string, newIDFunc func() string) (*feature, error) {
```

Comment:
The path argument is a filename here, does it make sense to rename it as well?

Reply:
as said, I just copied and did not much touch this code ...